app/client/components/identity/sharedStyles.ts

```diff
@@ -1,10 +1,22 @@
 import { CSSObject } from '@emotion/core';
+import { sans } from '../../styles/fonts';
 import palette from '../../colours';
 
 export const textSmall: CSSObject = {
 	fontSize: '14px',
 };
 
+export const standardSansText = {
+	fontFamily: sans,
+	fontSize: '14px',
+	lineHeight: '1.333',
+};
+
+export const toggleDescriptionPadding = {
+	margin: '0',
+	padding: '2.88px 90px 0 0',
```

Comment:
I imagine browsers would just round the 2.88 to 3?

Reply:
This was lifted from the existing style code and is used a couple other components I didn't touch. In light of upcoming UI overhaul leaving as is for now.